client.go

```diff
@@ -93,6 +93,7 @@ type Client interface {
 	GetNAS(ctx context.Context, id string) (NAS, error)
 	GetNASByName(ctx context.Context, name string) (NAS, error)
 	GetNfsServer(ctx context.Context, id string) (NFSServerInstance, error)
+	ListFS(ctx context.Context) ([]FileSystem, error)
```

Comment:
`GetFileSystems` would be a better name, but I see that we have used FS in other function names.

Reply:
I wanted to use the same name but in the other related methods, it was using `FS`, didn't wanted to disrupt the standardization. What would be your suggestion?